app/helpers/hyrax/collections_helper.rb

```diff
@@ -43,6 +43,29 @@ def collection_type_label(collection_type_gid)
       collection_type_gid
     end
 
+    def banner_file(collection_id)
```

Comment:
@jocyne requested 'Can this code go into a presenter rather than a Helper? Helpers tend to get cluttered and it's hard to see where the dependencies lie.'